File: Frontend/sopsc-mobile-app/src/types/report.ts
export interface Report {
  reportId: number;
  chaplain: string;
  chaplainDivision: string;
  hoursOfService?: number | null;
  commuteTime?: number | null;
  primaryAgency: string;
  typeOfService: string;
  contactName: string;
  pocPhone?: string | null;
  pocEmail?: string | null;
  clientName?: string | null;
  clientPhone?: string | null;
  addressDispatch: string;
  cityDispatch: string;
  addressDestination?: string | null;
  cityDestination?: string | null;
  dispatchTime?: string | null;
  arrivalTime?: string | null;
  milesDriven?: number | null;
  narrative: string;
  dateCreated: string;
  createdById: number;
}

File: Backend/SOPSC.Api/Models/Domains/Reports/Report.cs
using System;

namespace SOPSC.Api.Models.Domains.Reports
{
    /// <summary>
    /// Represents a chaplain service report.
    /// </summary>
    public class Report
    {
        public int ReportId { get; set; }
        public string Chaplain { get; set; }
        public string ChaplainDivision { get; set; }
        public decimal? HoursOfService { get; set; }
        public decimal? CommuteTime { get; set; }
        public string PrimaryAgency { get; set; }
        public string TypeOfService { get; set; }
        public string ContactName { get; set; }
        public string ContactPhone { get; set; }
        public string ContactEmail { get; set; }
        public string AddressDispatch { get; set; }
        public string CityDispatch { get; set; }
        public string Narrative { get; set; }
        public DateTime DateCreated { get; set; }
    }
}
